Add stubs to awu98 file

awu98 opened the pull request:
closes #10

awu98 committed:
Add stubs to awu98 file

awu98 committed:
Populated information :wq

willivolt commented on awu98-vancouver.md:
```diff
@@ -0,0 +1,19 @@
+#### Places to Eat
+- Sushi Ville
+- Marcello's
+- iHop
```
Mmm... Pancakes.

willivolt replied:
But what if I want doughnut :doughnut:?

awu98 committed:
Added to places to eat and deleted a place to go

Added 'East is East' to places to eat and deleted 'Steveston' from places to go.
